src/components/ResidentSearchForm.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import ResidentSearchForm from './ResidentSearchForm';
+
+test('it renders an input and button', async () => {
+  render(<ResidentSearchForm handleSearch={jest.fn()} />);
+  expect(screen.getByPlaceholderText('Search for a resident...'));
+  expect(screen.getByText('Search'));
+});
+
+test('it submits successfully', async () => {
```

Comment:
Mentioned in the other PR too, but Jest provides functions for better test descriptions. It's important so that running tests also act as documentation, as well as actual test assertions.

E.g. you could do 
```js
describe('Resident Search Form', () => {
  it('does something correctly')
})
```
so that when you run the tests, it's clear what the components are supposed to do, and when one fails, it's immediately obvious what has failed.